package.json

```diff
@@ -14,6 +14,8 @@
   },
   "devDependencies": {
     "@babel/core": "^7.7.0",
+    "@patternfly/react-core": "^4.84.4",
+    "@patternfly/react-table": "^4.19.45",
```

Comment:
Are these packaged?

Reply:
I think so :
https://github.com/theforeman/foreman-js/blob/master/packages/vendor-core/package.json#L32 
https://github.com/theforeman/foreman-js/blob/master/packages/vendor-core/package.json#L29

Reply:
You can remove both lines, those packages will be available via vendor which is already in peerDependencies.

Reply:
@xprazak2  done. thanks!